airflow/models/taskinstance.py

```diff
@@ -758,7 +758,7 @@ def generate_command(
     def log_url(self) -> str:
         """Log URL for TaskInstance."""
         iso = quote(self.execution_date.isoformat())
-        base_url = conf.get_mandatory_value("webserver", "BASE_URL")
+        base_url = conf.get_mandatory_value("webserver", "BASE_URL").rstrip("/")
```

Comment:
I feel like this could be lead to some errors in the future.

It's hard for someone to guess that retrieving the `base_url` from the config is a special case and needs to strip trailing '/'. (I can easily imagine someone doing `conf.get_mandatory_value("webserver", "BASE_URL")` to access the value) 

Maybe we can put that in the config accessor ? (So we don't need to bother, the equivalent of @validate for marshmallow field, that does so preprocesssing on the value before storing it)

Otherwise looking good

Reply:
I will check how can we do that for a specific config